ED: New, working ED integration test.

ashiklom opened the pull request:
This addresses #1329.

robkooper commented on tests/pecan64.ed.xml:
```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0"?>
 <pecan>
-  <outdir>pecan</outdir>
+  <outdir>/home/carya/pecan_tests/pecan64.ed.xml</outdir>
```
can you change this back to pecan (or pecan.ed) so it will work on any machine and not on just those that have a pecan_test folder as user carya.

ashiklom commented:
OK, I reverted to the EBI Farm test, added the C4 Grass PFT to the `pftmappings` file, and created a GitHub gist ([link](https://gist.github.com/ashiklom/20d6d9a2065f7d32cd5a593b62567773)) containing the revised CSS, PSS, and site files, which @robkooper will need to add to the VM.

julianpistorius commented:
> OK, I reverted to the EBI Farm test, added the C4 Grass PFT to the `pftmappings` file, and created a GitHub gist ([link](https://gist.github.com/ashiklom/20d6d9a2065f7d32cd5a593b62567773)) containing the revised CSS, PSS, and site files, which @robkooper will need to add to the VM.

While getting the latest ED2 (2.2.0) container working on UArizona HPC we ran into an issue with the ebifarm.lat40.0lon-88.0.site.* files downloaded from here: http://isda.ncsa.illinois.edu/~kooper/EBI/sites.tgz

In the logfile.txt:

```
+ Initializing from ED restart file. Node: 001
 [-] filelist_f: Checking prefix:  /data/sites/ebifarm/ebifarm.
                 Prefix length:       28
 +  Showing all     3 files:
   [-] File #:      1 /data/sites/ebifarm/ebifarm.lat40.0lon-88.0.css
   [-] File #:      2 /data/sites/ebifarm/ebifarm.lat40.0lon-88.0.pss
   [-] File #:      3 /data/sites/ebifarm/ebifarm.lat40.0lon-88.0.site
Using patch file: /data/sites/ebifarm/ebifarm.lat40.0lon-88.0.pss
 ISI           =       2
 CPOLY%NSITES  =       1
 IP            =       1
 CPOLY%SITENUM =       1
 Patch file    =  /data/sites/ebifarm/ebifarm.lat40.0lon-88.0.pss
 Site number  2002
